syntax = "proto3";

package test;

message TestMessage {
	int32 int_field = 1 ;
	bytes byte_field = 2;
}
